examples/knative/powershell/kn-ps-echo/Dockerfile

```diff
@@ -5,13 +5,14 @@ ENV PORT 8080
 # Set terminal. If we don't do this, weird readline things happen.
 RUN echo "/usr/bin/pwsh" >> /etc/shells && \
     echo "/bin/pwsh" >> /etc/shells && \
-    tdnf install -y powershell-7.0.3-2.ph3 unzip && \
+    tdnf install -y powershell icu openssl && \
```

Comment:
Why are we reverting to installing non-versioned packages?

Reply:
Fixed

Reply:
My comment was also related to the other components you install in that line. Currently there's two issues here:

- You can't easily bust the Docker build cache
- On a fresh environment (no cache) the build output could be different due to non-deterministic versions pulled

Reply:
Will fix per our meeting this morning